### ISSUE TITLE
Inconcistent labeling in popup.html

### ISSUE THREAD

**Issue by Jonta** [SimonBrazell/privacy-redirect#227]
https://github.com/SimonBrazell/privacy-redirect/blob/master/src/pages/popup/popup.html

- Nitter
- Invidious
- Bibliogram
- OpenStreetMap
- Reddit
- Search engine
- SimplyTranslate

Some of these are not like the others

"Search engine" (and maybe "SimplyTranslate"?) I can understand, if it redirects from more than Google, but "Reddit" sticks out as the only sensible one to me otherwise

"Nitter" is easy to guess as an alternative to "Twitter", but "Invidious"?

I suggest replacing the text with the main site it's replacing, instead giving us

- Twitter
- YouTube
- Instagram
- Google Maps
- Reddit
- Google search
- Google translate

=)

**Comment by Jonta** [comment]
@SimonBrazell Yes, no, maybe so? =)

**Comment by SimonBrazell** [comment]
Hi @Jonta nothing blocking this, just my own motivation and willingness to work on this project, it's a trivial change I just need to make time to implement it and push it out.

**Comment by SimonBrazell** [comment]
I agree with your suggestions, much more sensible naming, I think originally I was worried that the extension submission to the stores would be knocked back so I tried to make it less obvious to reviewers 🙂

**Comment by Jonta** [comment]
> Hi @Jonta nothing blocking this, just my own motivation and willingness to work on this project, it's a trivial change I just need to make time to implement it and push it out.

How can we help with that? =)

> I agree with your suggestions, much more sensible naming, I think originally I was worried that the extension submission to the stores would be knocked back so I tried to make it less obvious to reviewers slightly_smiling_face

Trademark issues?

Searching AMO now gives me

28 pages: Google maps
36 pages: Google search
08 pages: Google translate
16 pages: Instagram
36 pages: Reddit
69 pages: Twitter
13 pages: Wikipedia
90 pages: YouTube

**Comment by xanoni** [comment]
Agree, I had the same concern, this is why I didn't want to use the "brand names" in my two PRs here:

- https://github.com/SimonBrazell/privacy-redirect/pull/302
- https://github.com/SimonBrazell/privacy-redirect/pull/301
